pandas/core/common.py

```diff
@@ -233,6 +233,20 @@ def _any_none(*args):
     return False
 
 
+def _all_none(*args):
+    for arg in args:
+        if arg is not None:
+            return False
+    return True
```

Comment:
can you add a 1-liner doc-string to these, otherwise lgtm.

Reply:
Done and green.